chapters/compatibility.adoc

```diff
@@ -240,7 +240,11 @@ management.
 
 Media type versioning: Here, version information and media type are provided
 together via the HTTP Content-Type header — e.g.
-`application/x.zalando.cart+json;version=2`. For incompatible changes, a new
+
+TODO: dewayne.richardson
```

Comment:
json api has the following content-type which we use in all new apis: application/vnd.api+json